etc/upgrades/1711451944-hyperchain-upgrade/localhost/facets.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
revert? (and we'll create the migration separately)

Reply:
I can create the file manually if we want to delete it, it was easier until now